library/src/main/java/com/firebase/ui/auth/core/FirebaseResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.firebase.ui.auth.core;
+
+public class FirebaseResponse {
```

Comment:
Is there any way we can change this to an enum and use the ordinal for each value?